Dockerfile

```diff
@@ -8,7 +8,7 @@ FROM maven:${MAVEN_TAG} AS builder
 # the dependencies are changed and not the source code.
 COPY ./mb-solr/pom.xml mb-solr/pom.xml
 COPY ./mmd-schema/brainz-mmd2-jaxb/pom.xml brainz-mmd2-jaxb/pom.xml
-RUN cd brainz-mmd2-jaxb && \
+RUN --mount=type=cache,target=/root/.m2 cd brainz-mmd2-jaxb && \
```

Comment:
This is a syntax from buildkit, which caches the contents of `/root/.m2` between builds, allowing for faster maven commands.
This requires buildkit to be enabled, it's not enabled by default on linux. If this causes issues for building/deploying mbsolr we can remove it again